src/components/ColorPicker/components/SwatchBackground/SwatchBackground.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import {HSBColor} from '../../../../utilities/color-types';
+import {hsbToString} from '../../../../utilities/color-transformers';
+import styles from '../../ColorPicker.scss';
+
+export interface SwatchBackgroundProps {
+  color: HSBColor;
+}
+
+export function SwatchBackground(props: SwatchBackgroundProps) {
+  const {color} = props;
+
+  const style = {backgroundColor: hsbToString(color)};
+  return <div style={style} className={styles.SwatchBackground} />;
```

Comment:
Just curious why this is a component rather than inline?

Reply:
I think it can be inlined looking back at it 👍

I'll make the necessary changes